pkg/webhook/persistentpodstate/validating/persistent_create_update_handler.go

```diff
@@ -95,19 +105,17 @@ func validateUpdateObjImmutable(obj, old *appsv1alpha1.PersistentPodState, fldPa
 	return allErrs
 }
 
-func validatePersistentPodStateSpec(obj *appsv1alpha1.PersistentPodState, fldPath *field.Path) field.ErrorList {
+func validatePersistentPodStateSpec(obj *appsv1alpha1.PersistentPodState, fldPath *field.Path, whiteList *configuration.PPSWatchWatchCustomWorkloadWhiteList) field.ErrorList {
 	spec := &obj.Spec
 	allErrs := field.ErrorList{}
 	// targetRef
 	if spec.TargetReference.APIVersion == "" || spec.TargetReference.Name == "" || spec.TargetReference.Kind == "" {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("TargetReference"), spec.TargetReference, "empty TargetReference is not valid for PersistentPodState."))
 	}
-	_, err := schema.ParseGroupVersion(spec.TargetReference.APIVersion)
-	if err != nil {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("TargetReference"), spec.TargetReference, err.Error()))
-	}
-	if spec.TargetReference.Kind != "StatefulSet" {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("TargetReference"), spec.TargetReference, "TargetReference.Kind must be StatefulSet"))
+
+	apiVersion, kind := spec.TargetReference.APIVersion, spec.TargetReference.Kind
+	if spec.TargetReference.Kind != "StatefulSet" && !whiteList.ValidateAPIVersionAndKind(apiVersion, kind) {
```

Comment:
I think not the native or kruise StatefulSet, all other CRD(e.g. other StatefulSet) should judge whiteList.